packages/vuetify/src/composables/position.ts

```diff
@@ -0,0 +1,58 @@
+// Utilities
+import type { PropType } from 'vue'
+import { computed } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface PositionProps {
+  bottom?: boolean | string
+  left?: boolean | string
+  position?: 'static' | 'relative' | 'fixed' | 'absolute' | 'sticky'
+  right?: boolean | string
+  top?: boolean | string
```

Comment:
```suggestion
  bottom?: boolean | string | undefined
  left?: boolean | string | undefined
  position?: 'static' | 'relative' | 'fixed' | 'absolute' | 'sticky' | undefined
  right?: boolean | string | undefined
  top?: boolean | string | undefined
```

vue defines optional props as `foo?: type | undefined`

![image](https://user-images.githubusercontent.com/15625235/104820141-3a529880-5833-11eb-8970-56222cdd2284.png)


Also looks like there is `ExtractPropTypes` tool which can be used instead

Reply:
`something?: string` already includes `undefined`, that's probably just the IDE.

Reply:
ah ineed looks like it's IDE

Reply:
> `something?: string` already includes `undefined`, that's probably just the IDE.

This was my understanding as well.